text/0000-runtime-opts.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: runtime-opts
+- Start Date: 2019-11-02
+- RFC PR: 
+- Pony Issue: 
+
+# Summary
+
+This feature is a proposal to consolidate how we deal with these options, providing a way to avoid colissions with user defined options, upgrade and deprecation path across Pony releases. 
+
+# Motivation
+
+The options (`--pony*`) provided to a compiled Pony program to control its runtime environment have evolved overtime, with the latest instance being the `--ponythreads` to `--ponymaxthreads` [rename](https://github.com/ponylang/ponyc/issues/3318). With the current approach, this change would silently ignore the old option. It could possibly fail further parsing within user's application, as recognized `--pony*` arguments are _not_ exposed through `Env.args` the old, yet the now "unused" option would be exposed.
+The opposite being just as true: a user specified option could be elided if the runtime would consume it in a subsequent release. Nothing prohibits a user from using the `--pony` prefix. 
+
+# Detailed design
+
+Building upon [RFC-0064](0064-llvm-cli-opts-integration.md), as we probably should avoid introducing a new pattern, the proposal is to have a single option, namely `--pony_opts`, to pass all options as a single, comma seperated string, for the runtime to consume, e.g.:
```

Comment:
`--pony-opts` might be more idiomatic for CLI than `--pony_opts`?

That is, dashes are probably more idiomatic than underscores.